homework-submission/todo-app/src/todos/addform.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import DatePicker from 'react-datepicker';
+import moment from 'moment';
+ 
+import 'react-datepicker/dist/react-datepicker.css';
+import {inject, observer} from 'mobx-react';
+
+@inject('TodosStore')
+@observer
+
+class SendForm extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            description: '',
+            deadline: '',
+            done: false,
+            startDate: moment()
+        };
+    }
+
+    handleDateChange = date => {
+        this.setState({
+            deadline: date.format("DD-MM-YYYY"),
+            startDate: date
+        });
+    }
+
+    handleTextChange = e => {
+        this.setState({[e.target.name]: e.target.value});
+    }
+
+    handleSubmit = e => {
+        const {TodosStore} = this.props;
+        e.preventDefault();
+        const checkDateField = this.state.deadline;
+        if(checkDateField ==='') {
```

Comment:
what about `!checkDateField` ?
```suggestion
        if(!checkDateField) {
```

in other words you are using the `else TodosStore.addNewItem(this.state);` in line 43 again, so do we need the if statement? or we can solve it like the next suggestion may be?